src/replica/replica.h

```diff
@@ -625,6 +630,8 @@ class replica : public serverlet<replica>, public ref_counter, public replica_ba
     bool _is_manual_emergency_checkpointing{false};
     bool _is_duplication_master{false};
     bool _is_duplication_follower{false};
+    // replica is finding some private logs to load for duplication,avoid unexpected plog gc
```

Comment:
```suggestion
    // Indicate whether the replica is during finding out some private logs to
    // load for duplication. It useful to prevent plog GCed unexpectedly.
```